### PR TITLE
Fix for getting android getting device property for multiple instances

### PR DESCRIPTION
**Pull request by pearlselvan**
Fix for https://github.com/appium/appium/issues/5396

### CODE REVIEW

File: lib/adb.js [hunk 1]
```diff
@@ -193,6 +193,15 @@ ADB.prototype.shell = function (cmd, cb) {
   this.exec(execCmd, cb);
 };
 
+// This is fix for https://github.com/appium/appium/issues/5396 getting the devices propery when runnning with mulitiple appium instance
+ADB.prototype.shellForgettingDeviceProperty = function (cmd, cb) {
```
Comment (imurchie) [inline lib/adb.js]: `shellForgettingDeviceProperty` ought to be `shellForGettingDeviceProperty` (i.e., "for getting" is different than "forgetting").

File: lib/adb.js [hunk 2]
```diff
@@ -193,6 +193,15 @@ ADB.prototype.shell = function (cmd, cb) {
...
+  if (cmd.indexOf('"') === -1) {
+    cmd = '"' + cmd + '"';
+  }
+  var execCmd = '-s '+this.udid+' shell ' + cmd;
```
Comment (imurchie) [inline lib/adb.js]: Add spaces around the `+` signs.

**Comment by imurchie** [comment]
Thanks for this! A few style issues, otherwise it looks good.

**Comment by jlipps** [comment]
thanks @pearlselvan however i think we need to implement it in the 2.0 branch if we want it to be a part of appium moving forward.

**Comment by pearlselvan** [comment]
@jlipps , Thanks for your comments , could your please share is there any things should i take care to implement it in the 2.0 branch because this required to push for running appium in multiple real devices .

@imurchie : Thanks for your comment , currently i am working fixing your comment and will push in case if i get any further comment from @jlipps  regarding my above questions

**Comment by jlipps** [comment]
@pearlselvan can you try this with appium 1.5? i believe appium 1.5 already has fixed this issue because it always includes the `-s <id>` when running adb shell. if so then your fix won't be required there.

**Comment by imurchie** [comment]
All ADB calls have this built in now.

**Pull request closed without merging**